confignode/src/main/java/org/apache/iotdb/confignode/procedure/impl/pipe/task/DropPipeProcedureV2.java

```diff
@@ -130,12 +138,17 @@ public void serialize(DataOutputStream stream) throws IOException {
     stream.writeShort(ProcedureType.DROP_PIPE_PROCEDURE_V2.getTypeCode());
     super.serialize(stream);
     ReadWriteIOUtils.write(pipeName, stream);
+    pipeMeta.serialize(stream);
   }
 
   @Override
   public void deserialize(ByteBuffer byteBuffer) {
     super.deserialize(byteBuffer);
     pipeName = ReadWriteIOUtils.readString(byteBuffer);
+    try {
+      pipeMeta = PipeMeta.deserialize(byteBuffer);
+    } catch (IOException ignore) {
+    }
```

Comment:
This is sooo bad! Simply ignoring exceptions will be very dangerous. Why deserialize() in this class does NOT throw exceptions?

Reply:
Because the function it overrides does not throw it... And changing the overrided function cause more problems. However, if we do not use the "pipeMeta" as a member, this part can be ignored.